bitnami/mariadb-galera/templates/statefulset.yaml

```
@@ -8,7 +8,7 @@ spec:
   replicas: {{ .Values.replicaCount }}
   selector:
     matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
-  serviceName: {{ template "common.names.fullname" . }}
+  serviceName: {{ printf "%s-headless" (include "common.names.fullname" .) }}
```

It's always a good practice to ensure we meet the 63 characters limitation on service names (DNS limitation):
```suggestion
  serviceName: {{ printf "%s-headless" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" }}
```
Please apply the same at https://github.com/bitnami/charts/blob/master/bitnami/mariadb-galera/templates/headless-svc.yaml#L4

---

ok